client/src/components/PlayerInput.jsx

```diff
@@ -56,6 +56,13 @@ const PlayerInput = ({
             id={id}
             name={id}
             size='sm'
+            sx={{
+              '@media screen and (max-width: 991px)': {
```

Comment:
This looks MUCH cleaner! However, it only applies to PlayerInput button. A more elegant solution would be to apply this fix globally, so that every time we use a <Button> from ChakraUI, it'll include our modified functionality. The documentation link I sent you has this solution in it:

<img width="773" alt="Screenshot 2022-11-17 at 11 04 15 AM" src="https://user-images.githubusercontent.com/45023612/202523355-3d8c8dfe-61f5-437a-9bc6-06ce38749da7.png">

Then you can import that theme into client/src/index.js and pass it as a prop

<img width="414" alt="Screenshot 2022-11-17 at 11 05 22 AM" src="https://user-images.githubusercontent.com/45023612/202523515-3ea53594-b79a-46fa-a6bd-1d5b765de3d2.png">